src/app.py

```diff
@@ -3267,7 +3267,7 @@ def regen_mail_app_user_config_for_dovecot_and_postfix(only=None):
         if dovecot:
             hashed_password = _hash_user_password(settings["mail_pwd"])
             dovecot_passwd.append(
-                f"{app}:{hashed_password}::::::allow_nets=::1,127.0.0.1/24,local"
+                f"{app}:{hashed_password}::::::allow_nets=::1,127.0.0.1/24,local,mail={mail_user}@{mail_domain}"
             )
         if postfix:
             mail_user = settings.get("mail_user", app)
```

Comment:
(gotta move that line mail_user and the next one (mail_domain) prior to the 'if' block so it's available in the `if dovecot` too then)